migrations/2025-11-06-refactor-activities-to-nested-format.py

```diff
@@ -0,0 +1,245 @@
+#!/usr/bin/env python3
```

Comment:
```suggestion
#!/usr/bin/env -S uv run --script
```
might be better to be sure we are using the same venv as the rest of the project.